### ISSUE TITLE
[bitnami/kafka] KAFKA_CFG_MESSAGE_MAX_BYTES not respected for debian-12 images

### ISSUE THREAD

**Issue by Blaimi** [bitnami/containers#65494]
### Name and Version

bitnami/kafka:3.7.0-debian-12-r3

### What architecture are you using?

amd64

### What steps will reproduce the bug?

With the bitnami/kafka helm-chart configure the pod to have the environment-variable `KAFKA_CFG_MESSAGE_MAX_BYTES`:
```yaml
extraEnvVars:
  - name: KAFKA_CFG_MESSAGE_MAX_BYTES
    value: "50000000"
```

### What is the expected behavior?

The value gets respected

### What do you see instead?

The value gets not respected and is not visible in `/opt/bitnami/kafka/config/server.properties`

### Additional information

The issue was resolved for debian-11 images by #54458 but for debian-12 images the code was not copied: https://github.com/bitnami/containers/commit/823e01aa59dbf12c857c9930876601568657dd2e#diff-69514c5c223423a40a64533db25ab46e1d6a487048328ef8e0e9cc2aa8e2c16dR97

**Comment by carrodher** [comment]
Thank you for bringing this issue to our attention. We appreciate your involvement! If you're interested in contributing a solution, we welcome you to create a pull request. The Bitnami team is excited to review your submission and offer feedback. You can find the contributing guidelines [here](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md).

Your contribution will greatly benefit the community. Feel free to reach out if you have any questions or need assistance.

**Comment by Blaimi** [comment]
As it turned out, the problem was with the helm-chart configuration.

We migrated from an old version of the chart and didn't get, that the `extraEnvVars` `KAFKA_CFG_*` are not respected anymore. We switched to `extraConfig` and now it's working again.

The helm-chart creates the `server.properties` with a custom `kafka-init.sh` script. Because of this, the file already exists and the code from `libkafka.sh` is not applied.

This doesn't mean, that this problem doesn't exist in other environments. I ported the old commit to https://github.com/Blaimi/bitnami-containers/tree/fix/KAFKA_CFG_MESSAGE_MAX_BYTES during the investigation.

Shall I create a MR anyway? Otherwise feel free to close this issue.
